src/components/CommentsContainer/CommentsContainer.js

```diff
@@ -28,15 +29,24 @@ class CommentsContainer extends Component {
 
   render() {
     return (
-      <div className='comments-container'>
+      <> 
```

Comment:
this empty div doesnt seem necessary. The following section has wrong indentation

Reply:
Good point but in this case I need the fragment to hold my conditional rendering. Fixed indentation.